### DISCUSSION TITLE
Global config with window.Apex is not recognized with typescript

### DISCUSSION THREAD

**Discussion by ngtrthinh169TPC** [apexcharts/apexcharts.js#4094] [General]
As in title, typescript doesn't recognize the global variable of `Apex`.

I'm using Apexcharts with React-apexcharts. I know that I can just disable the error with ts-ignore or just type it as any, but that approaches are trying to ignore the problem, not solving it.

Any suggestions to make typescript happy here? I'd suggest we add something to d.ts file to cover this, but I don't really know if it can actually solve this.

**Comment by rashidzoft** [comment]
```
declare global {
  interface Window {
    Apex: {
      _chartInstances: ApexChart[];
    };
  }
}
```

Hope this will help
